### ISSUE TITLE
Dropped ctc_min_accuracy/coverage parameter

### ISSUE THREAD

**Issue by jdemeul** [nanoporetech/bonito#214]
I was wondering why the `ctc_min_accuracy` (and `ctc_min_coverage`) CLI parameter was dropped when generating training data and now seems to be fixed at 0.99 (and 0.9, respectively).

I'm analysing highly modified DNA (5mC and 6mA) which results in lower accuracy of the out-of-the-box basecalling models.
To increase accuracy I was refining the v3.1-3.3 models, and lowering the `ctc_min_accuracy` during training set generation to avoid biasing against highly modified (and hence more poorly basecalled) reads.
After retraining, the model has basically learned to ignore the modifications and does considerably better (example in the figure below).  

The issue is that with a fixed `ctc_min_accuracy` threshold at 0.99, only ~0.5% of my training data is written out and this data set is likely to be biased towards non-representative unmodified reads.

![GuppySUP-v33-v33refined-v33X2refined-denovo_alignmentaccuracy](https://user-images.githubusercontent.com/14054933/146164895-bf68b310-6c18-4530-9cee-5fcc3abbbc8d.png)

**Comment by andreaswallberg** [comment]
Dear @jdemeul did you manage to find a solution?

We are seeing coverage issues in training step resulting in few accepted reads used to build the model, and later on, short basecalled reads with our r10 data.

**Comment by jdemeul** [comment]
Hi @andreaswallberg, unfortunately I don't think anything has changed on this (even after the recent debacle of the released Dorado models performing poorly on bacterial DNA due to overtraining). 
I ended up just modifying it myself in the source code [here](https://github.com/nanoporetech/bonito/blob/b31229950a17459229a91c93aeca30fe67eb6ca8/bonito/io.py#L513-L514).

**Comment by andreaswallberg** [comment]
Hi @jdemeul , thanks for the feedback! Much appreciated.

Could you please indicate to values you changed those parameters?

Could you link to (or send a pm or email, I am sure you can find me) discussions regarding that bacterial basecalling? We are exploring the possibility that 0.4.3 models misbehave for us, so it would be good to get some extra hints here.

**Comment by davidnewman02** [comment]
`--min-accuracy-save-ctc` is still available in `bonito basecaller`: https://github.com/nanoporetech/bonito/blob/master/bonito/cli/basecaller.py#L216
